backend/tracim_backend/lib/core/workspace.py

```diff
@@ -257,6 +256,25 @@ def get_all_for_user(
         query = query.order_by(Workspace.label)
         return query.all()
 
+    def get_all_accessible_for_user(self, user: User) -> typing.List[Workspace]:
+        """
+        Return workspaces accessible by user.
+        Accessible workspaces:
+          - are of type OPEN or ON_REQUEST
+          - do not have user as a member
+        """
+        query = self._base_query_without_roles().filter(
```

Comment:
Can you add just a docstring to `_base_query_without_roles` method or rename it ? just checked, don't find this really explicit.

Reply:
Yes that's a good idea, I will do it.